docker/cypress-E2E-tests-mysql.yml

```diff
@@ -44,7 +44,9 @@ services:
         environment:
             CYPRESS_DB_TYPE: "MySQL"
             SORRY_CYPRESS_PROJECT_ID: "artemis-mysql"
-        command: sh -c "cd /app/artemis/src/test/cypress && chmod 777 /root && npm ci && npm run cypress:record:mysql"
+        command: >
+            sh -c "cd /app/artemis/src/test/cypress && chmod 777 /root && npm ci && npm run cypress:setup && npm run cypress:record:mysql & 
+            sleep 60 && cd /app/artemis/src/test/cypress && npm run cypress:record:mysql"
```

Comment:
```suggestion
            sh -c "cd /app/artemis/src/test/cypress && chmod 777 /root && npm ci && npm run cypress:setup && npm run cypress:record:mysql && 
            sleep 60 && cd /app/artemis/src/test/cypress && npm run cypress:record:mysql"
```
Mistake? Same below

Reply:
No that is actually what is doing the trick. All commands connected by `&&` are run in sequence and the `&` will sent the first command chain into the background. If we would use `&&` here, the first run would complexly finish and then it would start another run after that. This would double the run time instead of decreasing it 😅

Reply:
Ah, something new to learn ^^